### PR TITLE
Add desktop part

### PR DESCRIPTION
**Pull request by touredri**
In this pull request, we have 👍 

1. desktop part work add
2. navigation bar with portfolio, about, and contact
3. responsiveness to media queries

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @touredri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

## Required Changes ♻️
- [ ] Kindly make sure your page is responsive from the screen size of `375px` up to `1024px` as you may notice only the `1440px` looks similar to the original design.
`Hint: Kindly make sure you use the relative units(%) on your width instead of absolute(px)`

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -1,466 +1,1000 @@
...
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  * {
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+  }
+
```
Comment (Shedrack-Sunday) [inline style.css:496]: - [ ] Just as mentioned by the previous review I the review summary, please adjust your design to mimick the desktop version on the screen size of `768` and above. right now, your desktop version is only perfect on screens `1440` only. Remember your viewer's screens are not all the same, and this is an industry best practice to meet the various screen differences. 👍 

# Your view at `768`
image

# Expected 👍 
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @touredri    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing most of the features from the project requirements, However, there are a few you need to work on to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 



Required Changes ♻️

- [x] Firstly, you did have a very beautiful design(Atleast on screen size of `1440`). however, the goal is to stick to the recommended one just as it will be in your career as a developer. I've made a detailed review of sections that needs fixing. kindly follow the suggestions, and you'll be approved in no time. Excellent work you did 👏 

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ridwanediallo — COMMENTED** [review]
Hi @touredri , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linters. Please Fix the errors.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**